wpilibc/src/generated/main/native/include/frc/StadiaController.h

```diff
@@ -579,7 +584,6 @@ class StadiaController : public GenericHID {
    */
   [[deprecated("Use GetRightBumperButtonReleased instead")]]
   bool GetRightBumperReleased();
-
   /** Represents a digital button on an StadiaController. */
```

Comment:
Do we care about the whitespace change? It also happens on `XboxController.h`.

Reply:
It doesn't matter too much since it'll be removed next year, but the fix is to remove the `-` on the left side of the `elif` in the header template.